extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -109,6 +109,17 @@ export const profile = localize('profile', "Profile");
 export const selectConnection = localize('selectConnection', "Select connection");
 export const server = localize('server', "Server");
 export const defaultUser = localize('default', "default");
+export const selectProfile = localize('selectProfile', "Select publish profile to use");
```

Comment:
"load" instead of "use" for consistency with SSDT?

Reply:
Not commenting on "load" vs. "use", but overall: building out projects in ADS/VS Code is our opportunity to update strings if we think SSDT's wording could be improved or made more natural.